src/store/reducer.js

```diff
@@ -0,0 +1,15 @@
+import { createReducer, createAction } from '@reduxjs/toolkit';
+export const chooseState = createAction('choose_state');
+
+const initialState = {
+    stateCount: 0
+}
+
+const reducers = createReducer(initialState, (builder) => {
+    builder
+    .addCase(chooseState, (state, action) => {
+      return {...state, stateCount: action.payload}
+    })
+  });
+
+  export default reducers
```

Comment:
Missing semicolon.

There also should be an empty line in the end of the file!